### ISSUE TITLE
Fail to get extension lists.

### ISSUE THREAD

**Issue by KingMystic** [aniyomiorg/aniyomi#927]
### Steps to reproduce

1. Go to browse
2. Anime/Manga extension
3. Fail to get extension lists.

### Expected behavior

Anime and manga extension should be seen and can be installed, updated and can be installed.

### Actual behavior

Anime and manga extension can't be seen . And error comes out "Failed to get extension lists".

### Crash logs

--------- beginning of main
03-02 21:58:16.204 20970 20970 E ir.tachiyomi.m: Not starting debugger since process cannot load the jdwp agent.
03-02 21:58:16.426 20970 21019 E SQLiteLog: (284) automatic index on C(anime_id)
03-02 21:58:16.427 20970 21019 E SQLiteLog: (284) automatic index on animes_categories(anime_id)
App version: 0.12.3.10 (standard, 7722361, 72, 2023-02-09T10:47Z)
Android version: 12 (SDK 31)
Android build ID: SKQ1.211006.001 test-keys
Device brand: Xiaomi
Device manufacturer: Xiaomi
Device name: apollo
Device model: M2007J3SY
Device product name: apollo_global

### Aniyomi version

0.12.3.10

### Android version

Android 11

### Device

Mi 10T

### Other details

![Screenshot_2023-03-02-22-00-36-373_xyz jmir tachiyomi mi](https://user-images.githubusercontent.com/93214473/222499365-4f87db2f-85cb-4319-b856-53163f6024ee.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/jmir1/aniyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3.10](https://github.com/jmir1/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Manimdumb** [comment]
This is a problem because it would seem that the site and any possible sites related to it is blocked by your country.
I just found out myself, whenever i try to reload an ongoing anime series it just says connection reset

**Comment by Manimdumb** [comment]
Seems like the aniyomi app is going away forever
Nothing on it would work anymore

**Comment by KingMystic** [comment]
> > This is a problem because it would seem that the site and any possible sites related to it is blocked by your country. I just found out myself, whenever i try to reload an ongoing anime series it just says connection reset
> 
> I also have this problem, but it is not due to being blocked by a country. Firstly there are not blocks like that in the USA and secondly I have tried connecting from many different countries via VPN with no change.
> 
> Attempting to manually install the manga extensions also does not work.

I used the VPN but it is still not working .

**Comment by Helpful-Neko** [comment]
First time using this application and I kept running into the same issue

**Comment by KohicSimpx** [comment]
I have the same error, is the author going to release a patch?

**Comment by Anes-Temzini** [comment]
I have the same issue but I can still access the manga list of extensions that I downloaded.

**Comment by TCBWZA** [comment]
I doubt this is a country block (it works in Tachiyomi). 

I had an issue where the extension screen was behaving as above, I installed Tachiyomi and it showed that 9 of the extensions needed updates. I applied these through Tachi and now they have disappeared from Aniyomi. The items come up as unknown source in the migrate screen. 

If I try and install an extension manually it does not appear in the list either. (Download from Tachiyomi site and run the apk.)

The versions of extension that did not update still appear in the manga extensions list and seem to work. 

Did the extension registration process change in someway that now makes them incompatible?

If I can help isolate the issue please let me know.

The device is a samsung a8 tablet, factory standard running Android 13.

**Comment by TCBWZA** [comment]
I restored a backup from aniyomi into tachi it told me that the extensions that did not update were missing.

I restored a backup into tachi and then reviewed the migrate screen. All the updated extensions were there. The ones that did not update were missing from tachi, if I install them they then give me an update prompt.

Has tachi changed the permissions on the extensions, maybe?

**Comment by NessEvil** [comment]
I see that almost all of us have the same problem, I updated the extensions in tachi and now I can no longer access the manwhas that I had in aniyomi, does anyone know if there will be an update of the app?.
Thank you.

**Comment by reDpz** [comment]
> Seems like the aniyomi app is going away forever
> Nothing on it would work anymore 

wow really? really liked this app ive been using it for a few years idk what i will do without it. ill prob just stop watching anime all together

**Comment by 360pWatcher** [comment]
Same Here

**Comment by rahul6623** [comment]
Extension are not working 😔 unfortunately... hopefully it will be fixed soon enough.

**Comment by Lipexe** [comment]
please download the beta version posted on Discord

**Comment by TCBWZA** [comment]
> please download the beta version posted on Discord

I have installed the update on two devices and the extension list does seem to work now. I got Anime extensions updated, there were no Manga extension updates outstanding as I had already updated them through Tachi (mentioned above.)

PS: The new player looks good and the HW decode was active on my device (Samsung A8).

**Comment by battlewolf101** [comment]
> > please download the beta version posted on Discord
> 
> I have installed the update on two devices and the extension list does seem to work now. I got Anime extensions updated, there were no Manga extension updates outstanding as I had already updated them through Tachi (mentioned above.)
> 
> PS: The new player looks good and the HW decode was active on my device (Samsung A8). EDIT: I have successfully updates a Manga extension and the update applied correctly and is in sync with Tachi (both installed on the device.)

I am still unable to download any extensions of mangas, the anime extention work fine but unable to download any manga extension and aniyomi was my only way for reading manga please help and tell the whole process you had to go through to get it to work. I mean i even downloaded tachiyomi so that after i downloaded extensions in tachiyomi it will show up in aniyomi too but it didnt.

**Comment by TCBWZA** [comment]
> > > please download the beta version posted on Discord
> > 
> > 
> > I have installed the update on two devices and the extension list does seem to work now. I got Anime extensions updated, there were no Manga extension updates outstanding as I had already updated them through Tachi (mentioned above.)
> > PS: The new player looks good and the HW decode was active on my device (Samsung A8). EDIT: I have successfully updates a Manga extension and the update applied correctly and is in sync with Tachi (both installed on the device.)
> 
> I am still unable to download any extensions of mangas, the anime extention work fine but unable to download any manga extension and aniyomi was my only way for reading manga please help and tell the whole process you had to go through to get it to work. I mean i even downloaded tachiyomi so that after i downloaded extensions in tachiyomi it will show up in aniyomi too but it didnt.

I found the last 2 betas have had issues, I have rolled back each time as others have complained I did not report anything. I had it working on r6141. Hope this helps.

**Comment by 12335666** [comment]
> I see that almost all of us have the same problem, I updated the extensions in tachi and now I can no longer access the manwhas that I had in aniyomi, does anyone know if there will be an update of the app?.
> Thank you.

Well you could try https://github.com/jmir1/aniyomi-mpv-beta/releases/tag/r6141
This version of aniyomi has fixed the extension problem

**Comment by 12335666** [comment]
https://github.com/jmir1/aniyomi-mpv-beta/releases/tag/r6141
This version of aniyomi has fixed the extension problem

**Comment by Quickdesh** [comment]
Duplicate of #844
